.github/workflows/zcashd-nightly.yml

```
@@ -0,0 +1,97 @@
+name: zcashd-nightly
+
+on:
+  workflow_dispatch:    
+  push:
+    branches: [ "zcash-ci" ]
```

```suggestion
    branches: [ "main" ]
```

I think `zcash-ci` won't exist after this PR is merged

---

I assumed we'd have a separate branch dedicated to collecting these results, but if not, then 👆